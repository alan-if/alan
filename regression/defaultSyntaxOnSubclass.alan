Every vehicle Isa object
End Every vehicle.

Every car Isa vehicle
  Verb drive
    Does "Yoooohooooo!"
  End Verb.
End Every car.

Every bus Isa vehicle
End Every bus.

The car1 Isa car At l
End The car1.

The bus1 Isa bus At l
End The bus1.

The l Isa location
End The l.

Start At l.
