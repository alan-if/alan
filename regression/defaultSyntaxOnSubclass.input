drive bus1
drive car1
