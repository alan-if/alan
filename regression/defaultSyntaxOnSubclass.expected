########## defaultSyntaxOnSubclass ##########

        No warnings or errors detected.
        1 informational message(s).



L
There is a car1 and a bus1 here.

> drive bus1
You can't do that.

> drive car1
Yoooohooooo!

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
